templates/layout/_header.html.twig

```diff
@@ -150,7 +150,7 @@
                     <ul class="dropdown__menu">
                         <li>
                             <a href="{{ path('user_overview', {username: app.user.username}) }}"
-                               class="{{ html_classes({'active': is_route_name_contains('user_overview')}) }}">
+                               class="{{ html_classes({'active': is_route_name_contains('user_overview') and user is same as app.user}) }}">
```

Comment:
I swear I browsed and posted things and everything... if this ever 500s and says "user is undefined"... maybe I should add `and user is defined` _just_ in case... but it didn't seem needed even after using my site a bunch... maybe I should double check in app_env prod just to make sure, wouldn't want this to cause any issues